fetch/main.js

```diff
@@ -0,0 +1,6 @@
+async function logData() {
+  const response = await fetch('https://jsonplaceholder.typicode.com/users');
+  const data = await response.json();
```

Comment:
Check response.ok before using response.json(). If ok is false then response.json() might not exist. The best thing to do is to throw a new Error if response.ok is false.